pkg/plugins/mco/mco_plugin.go

```diff
@@ -0,0 +1,201 @@
+package main
```

Comment:
The same code will be used for vanilla k8s, can you rename it to switchdev_plugin, code here is only related to switchdev, it doesn't recognize how the service injected

Reply:
This plugin does 2 tasks: 1) generate switchdev.conf file; 2) move the node to an MCP. The second task is only for MCO. I think for vanilla k8s, we can create a separte plugin to do 1) provision the swichdev.configuratio.service; 2) generate switchdev.conf file. And we can load plugins based on the ClusterType. The logic of generating swichdev.conf can be reused in both plugins. But I think it can be addressed later.